src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,266 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "Blazor",
+    "WebAssembly",
+    "PWA",
+    "Empty"
+  ],
+  "name": "Blazor WebAssembly App Empty",
+  "defaultName": "BlazorApp-empty",
+  "description": "An empty project template for creating a Blazor app that runs on WebAssembly and is optionally hosted by an ASP.NET Core app. This template does not have any content in it.",
+  "groupIdentity": "Microsoft.Web.Blazor.Wasm.Empty",
+  "precedence": "9000",
+  "identity": "Microsoft.Web.Blazor.Wasm.Empty.CSharp.7.0",
+  "thirdPartyNotices": "https://aka.ms/aspnetcore/7.0-third-party-notices",
+  "preferNameDirectory": true,
+  "primaryOutputs": [
+    {
+      "condition": "(!Hosted)",
+      "path": "EmptyComponentsWebAssembly-CSharp.csproj"
+    },
+    {
+      "condition": "(Hosted && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "Client/EmptyComponentsWebAssembly-CSharp.Client.csproj"
+    },
+    {
+      "condition": "(Hosted && HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "path": "Shared/EmptyComponentsWebAssembly-CSharp.Shared.csproj"
+    }
+  ],
+  "shortName": "blazorwasm-empty",
+  "sourceName": "EmptyComponentsWebAssembly-CSharp",
+  "sources": [
+    {
+      "source": "./",
+      "target": "./",
+      "exclude": [
+        ".template.config/**"
+      ],
+      "copyOnly": [
+        "**/wwwroot/css/**"
+      ],
+      "modifiers": [
+        {
+          "condition": "(!Hosted)",
+          "exclude": [
+            "Server/**",
+            "Shared/**",
+            "*.sln"
+          ],
+          "rename": {
+            ".Client.csproj": ".csproj",
+            "Client": "."
+          }
+        },
+        {
+          "condition": "(Hosted)",
+          "exclude": [
+            "Client/wwwroot/sample-data/**"
+          ]
+        },
```

Comment:
Do we still need this? Looks like you _may_ have removed the `sample-data` directory?

Reply:
Yes, this is not needed. This directory has been deleted.